internal/logging/logging.go

```diff
@@ -67,54 +56,74 @@ func NewLogWithWriters(stdout, stderr io.Writer, opts ...func(*LogWithWriters))
 	return lw
 }
 
-func (lw *LogWithWriters) WriterForLevel(level logging.Level) io.Writer {
-	if lw.Level > log.Level(level) {
-		return ioutil.Discard
+// WithClock is an option used to initialize a LogWithWriters with a given clock function
+func WithClock(clock func() time.Time) func(writers *LogWithWriters) {
+	return func(logger *LogWithWriters) {
+		logger.clock = clock
 	}
+}
 
-	if level == logging.ErrorLevel {
-		return lw.errOut
+// WithVerbose is an option used to initialize a LogWithWriters with Verbose turned on
+func WithVerbose() func(writers *LogWithWriters) {
+	return func(logger *LogWithWriters) {
+		logger.Level = log.DebugLevel
 	}
-
-	return lw.out
 }
 
+// HandleLog handles log events, printing entries appropriately
 func (lw *LogWithWriters) HandleLog(e *log.Entry) error {
 	lw.Lock()
 	defer lw.Unlock()
 
 	writer := lw.WriterForLevel(logging.Level(e.Level))
-	if lw.wantTime {
-		ts := lw.clock().Format(timeFmt)
-		_, _ = fmt.Fprint(writer, appendMissingLineFeed(fmt.Sprintf("%s %s%s", ts, formatLevel(e.Level), e.Message)))
-		return nil
+	_, err := fmt.Fprint(writer, appendMissingLineFeed(fmt.Sprintf("%s%s", formatLevel(e.Level), e.Message)))
+
+	return err
+}
+
+// WriterForLevel returns a Writer for the given logging.Level
+func (lw *LogWithWriters) WriterForLevel(level logging.Level) io.Writer {
```

Comment:
Since it's a public function and you changed its return value in some cases, just wanted to be sure that you checked that no one else, except `HandleLog` (and some tests) is using it.

Reply:
It's a public func, but inside `internal`, so I'm not worried about external users. We do use WriterForLevel in a few places, but the change only added time to the error message, which seemed like a worthwhile goal.